doc/modules/ROOT/pages/index.adoc

```diff
@@ -128,7 +456,51 @@ image::ss5.5-create-instance.png[]
 
 As the last step, select your https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[key pair]
 for the instances and that's it. You can launch instances now.
+--
+
+{tab-header-CLI}::
++
+--
+Using the id's and the names you set above, now run two EC2 instances. In addition to the steps so far, you need to have
+a https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html[key pair] and pass it via `--key-name` flag.
+The command below will start two Amazon Linux machines with type t2.micro and assign public IP addresses to each.
+Also a tag named "cluster-tag" with "guide-ec2-cluster" value will be assigned to the instances:
 
+[source, shell]
+----
+$ aws ec2 run-instances --image-id ami-04d29b6f966df1537 \
+    --count 2 --instance-type t2.micro \
+    --key-name *********** \
+    --security-group-ids sg-*********** \
+    --subnet-id subnet-*********** \
+    --iam-instance-profile Name="hazelcast-guide-EC2-Instance-Profile" \
+    --associate-public-ip-address \
+    --tag-specifications 'ResourceType=instance,Tags=[{Key=cluster-tag,Value=guide-ec2-cluster}]'
+----
+
+Then fetch the instance id's filtered by their tags:
+[source, shell]
+----
+$ aws ec2 describe-instances \
+    --filters "Name=tag:cluster-tag,Values=guide-ec2-cluster" \
+    --query "Reservations[].Instances[].InstanceId"
+
+- i-xxxxxxxxxxxxxxxxx
+- i-yyyyyyyyyyyyyyyyy
+----
+
+Finally, fetch the public IPs of the instances to establish SSH connections in the next step:
+[source, shell]
+----
+$ aws ec2 describe-instances \
+    --instance-ids i-xxxxxxxxxxxxxxxxx i-yyyyyyyyyyyyyyyyy \
+    | grep PublicIpAddress
```

Comment:
An alternative to grep is `--query 'Reservations[*].Instances[*].PublicIpAddress'`. Up to you.

Reply:
I was looking for that, will update accordingly. Thanks.